pkg/controllers/health/controller.go

```diff
@@ -0,0 +1,17 @@
+package health
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func NewHealthController(router *gin.Engine) {
+	router.GET("/health", HealthHandler)
+}
+
+func HealthHandler(gctx *gin.Context) {
+	gctx.Header("Access-Control-Allow-Origin", "*")
```

Comment:
I see we use these headers in multiple places. Maybe it's a good idea to move these to middleware?

Reply:
moved this to middleware.